gcloud/streaming/http_wrapper.py

```diff
@@ -157,7 +173,7 @@ def length(self):
         Returns:
           Response length (as int or long)
         """
-        def ProcessContentRange(content_range):
+        def _process_content_range(content_range):
```

Comment:
Is a closure needed? I'd rather use a method or instances of a `__call__`-able class.